docs/howtos/integrations/_langgraph_agent_evaluation.md

```diff
@@ -275,7 +275,7 @@ Ragas uses its own format to evaluate agent interactions. So, if you're using La
 
 **Goal:**  Convert the list of LangChain messages (e.g., HumanMessage, AIMessage, and ToolMessage) into the format expected by Ragas, so the evaluation framework can understand and process them properly.
 
-To convert a list of LangChain messages into a format suitable for Ragas evaluation, Ragas provides the function convert_to_ragas_messages, which can be used to transform LangChain messages into the format expected by Ragas.
+To convert a list of LangChain messages into a format suitable for Ragas evaluation, Ragas provides the function [convert_to_ragas_messages](ragas.integrations.langgraph), which can be used to transform LangChain messages into the format expected by Ragas.
```

Comment:
```suggestion
To convert a list of LangChain messages into a format suitable for Ragas evaluation, Ragas provides the function [convert_to_ragas_messages][ragas.integrations.langgraph.convert_to_ragas_messages], which can be used to transform LangChain messages into the format expected by Ragas.
```

need to be exact path to method (I think - will check in the rendered docs) and surrounded by square brackets

Reply:
Should I commit the changes or wait for you confirmation? similar change would be required in md cell of ipynb file as well, right?

Reply:
I'm looking to see if it worked - the corrent link doesn't https://ragas--1636.org.readthedocs.build/en/1636/howtos/integrations/_langgraph_agent_evaluation/#converting-messages-to-ragas-evaluation-format

Reply:
can you check it locally? and then make the change? I know you have to use square brackets , not sure about using the full path (I feel that would be better)

Reply:
Are you sure about the using square brackets for both because it not linking and becomes normal text. I used something like this and it is linking but sot sure if it is the right way. I also had a look in the previous commits where something like [this](https://github.com/explodinggradients/ragas/commit/4cfa6b177bc66f6050ddbefbf43b94df8832fd8b#diff-9f80f0d8beb02e6e5931d61f3640ca25cd0b0a16daf898ee4c00ab83ce57782bR80) is used.

[convert_to_ragas_messages]\(../../../src/ragas/integrations/langgraph.py\)

Reply:
I'm very sure about the square brackets 
![image](https://github.com/user-attachments/assets/25d91ec8-c1bf-45df-a3d6-c853831c11a6)
![image](https://github.com/user-attachments/assets/3add68db-8f07-43ee-b232-6136bfa0949b)
you can easily verify this locally

https://github.com/explodinggradients/ragas/blob/main/docs/getstarted/rag_testset_generation.md